models/decks-models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+const { Deck } = require("../db/models/decks");
+
+exports.fetchDecks = async () => {
+  return await Deck.find({}).then((decks) => {
```

Comment:
Are we keeping the await / async way of handling async functions ? or are we keeping promise chains ?